src/ingest_validation_tools/docs_utils.py

```diff
@@ -377,30 +377,54 @@ def _make_toc(md):
     return f'<blockquote markdown="1">\n\n{joined_mds}\n\n</blockquote>\n'
 
 
-def _make_dir_description(dir_schema):
+def _make_dir_descriptions(dir_schemas):
+    sorted_items = sorted(dir_schemas.items(), key=lambda item: item[0])
```

Comment:
Minor nitpick: might be worth implementing as
```
sorted_items = sorted(dir_schemas.items(), key=operator.itemgetter(0), reverse=True)
```

(or `key=itemgetter(0)` after `from operator import itemgetter` at the beginning of the file)